### ISSUE TITLE
effectiveness of gradient

### ISSUE THREAD

**Issue by gradinnovate** [alga-hopf/dl-spectral-graph-partitioning#2]
Hi,

I noticed that Graclus clustering is applied between the GNN layers. Is that suitable for backpropagation?

**Comment by alga-hopf** [comment]
Hi, yes it should be suitable for backpropagation. Here is the torch_cluster implementation
https://github.com/rusty1s/pytorch_cluster/blob/master/torch_cluster/graclus.py
